src/extension/background-script/connectors/connector.interface.ts

```diff
@@ -13,17 +14,17 @@ interface Route {
 
 export interface ConnectorInvoice {
   custom_records?: {
-    "696969": string;
-    "7629169": string;
-    "5482373484": string;
-  };
+    "696969"?: string;
+    "7629169"?: string;
+    "5482373484"?: string;
+  } & Record<string, string>;
   id: string;
   memo: string;
   preimage: string;
   settled: boolean;
   settleDate: number;
   totalAmount: string;
-  type: "received";
+  type: string;
```

Comment:
undo - this should be type: "received"

Reply:
oauth connector will be using type as received instead of incoming/outgoing. resolved